tgui/packages/tgui/styles/components/Section.scss

```
@@ -57,26 +61,39 @@ $shadow-offset: 0 0 !default;
   height: 100%;
 }
 
-.Section--fill .Section__content {
+.Section--fill .Section__rest {
   flex-grow: 1;
 }
 
+.Section--fill.Section--scrollable .Section__content {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+}
+
 .Section--iefix.Section--fill {
   display: table !important;
   width: 100% !important;
   height: 100% !important;
   border-collapse: collapse;
   border-spacing: 0;
 
-  & .Section__content {
+  & .Section__rest {
     display: table-row !important;
     height: 100% !important;
   }
 }
 
 .Section--scrollable {
   overflow-x: hidden;
+  overflow-y: hidden;
+}
+
+.Section--scrollable .Section__content {
   overflow-y: scroll;
+  overflow-x: hidden;
 }
```

Not taking advantage of SCSS nesting here?
```suggestion
.Section--scrollable {
  overflow-x: hidden;
  overflow-y: hidden;

  .Section__content {
    overflow-y: scroll;
    overflow-x: hidden;
  }
}
```